parm/fv3lam_rrfs.xml

```diff
@@ -727,11 +727,36 @@
        </param>
 
        <param>
-       <shortname>ACM_WEASD_ON_SURFACE</shortname>
-       <pname>WEASD</pname>
+       <shortname>ACM_SNOWFALL_ON_SURFACE</shortname>
+       <pname>TSNOWP</pname>
        <scale>4.0</scale>
        </param>
 
+       <param>
+       <shortname>ACM_GRAUPEL_ON_SURFACE</shortname>
+       <scale>6.0</scale>
+       </param>
+
+       <param>
+       <shortname>ACM_FRAIN_ON_SURFACE</shortname>
+       <scale>6.0</scale>
+       </param>
+
+       <param>
+       <shortname>BUCKET_SNOWFALL_ON_SURFACE</shortname>
+       <scale>6.0</scale>
+       </param>
+
+       <param>
+       <shortname>BUCKET_GRAUPEL_ON_SURFACE</shortname>
+       <scale>6.0</scale>
+       </param>
+
+       <param>
+       <shortname>BUCKET_FRAIN_ON_SURFACE</shortname>
+       <scale>6.0</scale>
+       </param>
+
```

Comment:
@EricJames-NOAA These variables were already added with @ericaligo-NOAA 's PR #600

Reply:
These are the fields we have in fv3lam_rrfs.xml
 681        <param>
 682        <shortname>ACM_GRAUPEL_ON_SURFACE</shortname>
 683        <scale>6.0</scale>
 684        </param>
 685
 686        <param>
 687        <shortname>BUCKET_GRAUPEL_ON_SURFACE</shortname>
 688        <scale>6.0</scale>
 689        </param>
 690
 691        <param>
 692        <shortname>ACM_FRAIN_ON_SURFACE</shortname>
 693        <scale>6.0</scale>
 694        </param>
 695
 696        <param>
 697        <shortname>BUCKET_FRAIN_ON_SURFACE</shortname>
 698        <scale>6.0</scale>
 699        </param>
 700
 701        <param>
 702        <shortname>ACM_SNOWFALL_ON_SURFACE</shortname>
 703        <scale>6.0</scale>
 704        </param>
 705
 706        <param>
 707        <shortname>BUCKET_SNOWFALL_ON_SURFACE</shortname>
 708        <scale>6.0</scale>
 709        </param>